examples/client-compilation.jade

```diff
@@ -0,0 +1,140 @@
+//
+  This example shows how to compile and render Jade templates in the browser.
+
+  Note that although it is supported, we do not *recommend* compiling in the
+  browser; rather, you should pre-compile the template using compileClient
+  or compileFileClient.
+
+//-
+  You can either compile this to examples/client-compilation.html and open it
+  manually with a web browser, or you can serve this file using the server
+  examples/client-compilation.js.
+
+doctype html
+html
+  head
+    title Jade compilation demo
+    style.
+      * {
+        box-sizing: border-box;
+      }
+      iframe, pre, textarea {
+        height: auto;
+        border: 1px solid #a9a9a9;
+        width: 100%;
+        max-height: 600px;
+        min-height: 120px;
+      }
+      /* Preformatted <textarea> */
+      textarea.pre {
+        font-family: monospace;
+        font-size: 1em;
+      }
+      pre {
+        padding: 2px;
+        overflow: auto;
+        width: auto;
+      }
+      /* Prettier button (half-)stolen from Bootstrap */
+      button {
+        padding: 8px 16px;
+        font-size: 19px;
+        border: 1px solid transparent;
+        border-radius: 5px;
+        background-color: #fff;
+        border-color: #ccc;
+        cursor: pointer;
+      }
+      button:hover,
+      button:focus,
+      button:active {
+        color: #333;
+        background-color: #e6e6e6;
+        border-color: #adadad;
+      }
+
+  body(onload='comp()')
+    h1: button(type='submit', onclick='comp()') Submit
+
+    h2 Input template
+    textarea#input.pre(rows='8').
+      doctype html
+
+      html
+        body
+          p I love you.
+          p: small From \#[em= name].
+
+    h2 Compilation options (JSON)
+    textarea#options.pre(rows='8').
+      {
+        "pretty": true
+      }
+
+    h2 Locals (JSON)
+    textarea#locals.pre(rows='8').
+      {
+        "name": "Timothy"
+      }
+
+    h2 Raw HTML
+    pre#output
+
+    h2 Rendered HTML
+    iframe#output-html
+
+    script(src='../jade.js')
```

Comment:
Yes, it is not the correct path. I did this because of two reasons:

1. it will work if the user render the template to examples/client-compilation.html
2. it will work even in a server environment because the page is served as `http://127.0.0.1/`, and its `../jade.js` is rendered as `http://127.0.0.1/jade.js` (because there is no higher level). I agree this is a hack however.